stable/datadog/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: datadog
-version: 1.2.0
+version: 1.2.1
```

Comment:
since this is a new feature, I would bump to `1.3.0`

Reply:
+1 this is not a bug fix, it deserves a minor version bump.